apiserver/Makefile

```diff
@@ -137,11 +136,6 @@ $(CONTAINER_FIPS_CREATED): docker-image/Dockerfile.$(ARCH) $(BINDIR)/apiserver-$
 lint-cache-dir:
 	mkdir -p $(CURDIR)/.lint-cache
 
-check-boring-ssl: $(BINDIR)/apiserver-$(ARCH)
```

Comment:
This is part of `build_cgo_boring_binary` helper function in lib.Makefile. No need a separate target to validate.